crates/extensions/tedge_downloader_ext/src/builder.rs

```diff
@@ -0,0 +1,50 @@
+use crate::DownloadRequest;
+use crate::DownloadResponse;
+use crate::DownloaderActor;
+use tedge_actors::Builder;
+use tedge_actors::DynSender;
+use tedge_actors::LinkError;
+use tedge_actors::NoConfig;
+use tedge_actors::RuntimeRequest;
+use tedge_actors::RuntimeRequestSink;
+use tedge_actors::ServiceConsumer;
+use tedge_actors::ServiceProvider;
+use tedge_actors::SimpleMessageBoxBuilder;
+
+pub struct DownloaderBuilder {
+    box_builder: SimpleMessageBoxBuilder<DownloadRequest, DownloadResponse>,
+}
+
+impl DownloaderBuilder {
+    pub fn default() -> Self {
+        DownloaderBuilder {
+            box_builder: SimpleMessageBoxBuilder::new("DownloaderActor", 16),
+        }
+    }
+}
+
+impl RuntimeRequestSink for DownloaderBuilder {
+    fn get_signal_sender(&self) -> DynSender<RuntimeRequest> {
+        Box::new(self.box_builder.get_signal_sender())
```

Comment:
```suggestion
        self.box_builder.get_signal_sender()
```

Reply:
This file itself got removed since I changed from `Actor` to `Server`.